gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -339,6 +342,11 @@ MM_VerboseHandlerOutput::outputInitializedStanza(MM_EnvironmentBase *env, MM_Ver
 #endif /* OMR_GC_MODRON_SCAVENGER */
 	buffer->formatAndOutput(env, 1, "<attribute name=\"splitFreeListSplitAmount\" value=\"%zu\" />", _extensions->splitFreeListSplitAmount);
 	buffer->formatAndOutput(env, 1, "<attribute name=\"numaNodes\" value=\"%zu\" />", _extensions->_numaManager.getAffinityLeaderCount());
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	if (_extensions->reinitalizationInProgress()) {
+		buffer->formatAndOutput(env, 1, "<attribute name=\"CRIU Restore\" value=\"%s\" />", "true");
```

Comment:
don't use CRIU, but something more generic

Reply:
Changed to `Restored Snapshot`